heare/developer/compacter.py

```diff
@@ -65,6 +65,10 @@ def __init__(
 
     def count_tokens(self, messages: List[MessageParam], model: str) -> int:
```

Comment:
Don't keep around deprecated code, use the correct implementation and fix all the call sites to the new method signature.